app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionController.scala

```diff
@@ -82,6 +85,14 @@ class SubscriptionController @Inject()(
 
   val desAddressForm = new DesAddressForm(Logger, blacklistedPostCodes)
 
+  private val linkAccountForm: Form[LinkAccount] =
+    Form[LinkAccount](
+      mapping("autoMapping" -> optional(text).verifying(FieldMappings.radioInputSelected))(ans =>
```

Comment:
May be we can avoid using optional(text) here as autoMapping always returns yes/no.

Reply:
I think we need the optional, just so we can return a valid error "no-radio-selected" if submitting nothing